gammapy/estimators/map/core.py

```diff
@@ -870,6 +896,9 @@ def to_hdulist(self, sed_type="likelihood", hdu_bands=None):
         hdulist : `~astropy.io.fits.HDUList`
             Map dataset list of HDUs.
         """
+        if sed_type is None:
+            sed_type = self.sed_type_init
```

Comment:
maybe test the default behavior?

Reply:
I changed the to `sed_type=None` in the signature, so this behavior is actually tested in many places.